tests/test_scene_rendering/test_cli_flags.py

```diff
@@ -164,3 +164,27 @@ def test_custom_folders(tmp_path, manim_cfg_file, simple_scenes_path):
 
     exists = (tmp_path / "SquareToCircle.png").exists()
     assert exists, "--custom_folders did not produce the output file"
+
+
+@pytest.mark.slow
+def test_dash_as_filename(tmp_path):
+    code = "class Test(Scene):\n    def construct(self):\n        self.add(Circle())\n        self.wait(1)"
+    command = " ".join(
+        [
+            "echo",
+            f'"{code}"',
```

Comment:
*Pretty* sure this doesn't need to be put in quotation marks, so
```suggestion
            code,
```

Reply:
Just tried it on my system. It throws a syntax error:

```sh
>       assert exit_code == 0, err
E       AssertionError: /bin/sh: -c: line 0: syntax error near unexpected token `('
E         /bin/sh: -c: line 0: `echo class Test(Scene):'
E         
E       assert 2 == 0

```

Reply:
Ah yep, fair enough then